### ISSUE TITLE
SQL options are being ignored

### ISSUE THREAD

**Issue by dermoth** [bitcoin-abe/bitcoin-abe#178]
@jtobey , It appears with the addition of SqlAbstration.py a lot of SQL options are now being ignored, failing back to auto-detection.

Are we supposed to deprecate all the forced options, such ```--binary-type``` and ```--int-type```? If not, we'll need to patch them all in a way similar to this (and I'd like to know if you'rd ok importing the whole args into sql or only the specific options we care about...)

```
diff --git a/Abe/SqlAbstraction.py b/Abe/SqlAbstraction.py
index 90049e0..517e8af 100644
--- a/Abe/SqlAbstraction.py
+++ b/Abe/SqlAbstraction.py
@@ -35,6 +35,7 @@ class SqlAbstraction(object):
         sql.connect_args = args.connect_args
         sql.prefix = args.prefix
         sql.config = args.config
+        sql.args = args
 
         sql.log    = logging.getLogger(__name__)
         sql.sqllog = logging.getLogger(__name__ + ".sql")
@@ -642,7 +643,10 @@ class SqlAbstraction(object):
             "Binary type " + tests[0] + " fails test")
 
     def configure_int_type(sql):
-        defaults = ['int', 'decimal', 'str']
+        defaults = (['int', 'decimal', 'str']
+            if sql.args.int_type is None else
+            [ sql.args.int_type ])
+
         tests = (defaults if sql.config.get('int_type') is None else
                  [ sql.config['int_type'] ])
 
```

**Comment by jtobey** [comment]
@dermoth, use your judgment.  It looks like a bug to me, and I don't have time to debug it.  The same goes for the other issues you raised.  Thanks.  I have made you an owner of bitcoin-abe.

**Comment by dermoth** [comment]
Fixed by e350ed2 -- I restored the config option's behavior.
